pkgs/top-level/all-packages.nix

```diff
@@ -13158,7 +13160,9 @@ with pkgs;
 
   # A function to build a manually-configured kernel
   linuxManualConfig = pkgs.buildLinux;
-  buildLinux = makeOverridable (callPackage ../os-specific/linux/kernel/manual-config.nix {});
+  buildLinux = makeOverridable (callPackage ../os-specific/linux/kernel/manual-config.nix {
+    buildPackages = buildPackages // { stdenv = overrideCC buildPackages.stdenv gcc7; };
```

Comment:
Please add a comment here, why gcc7 is required.
This makes it easier to remove this line later, when gcc7 becomes default or is superseded by newer versions.

Reply:
Added a comment.